lib/igniter/project/application.ex

```diff
@@ -78,6 +78,13 @@ defmodule Igniter.Project.Application do
     end
   end
 
+  @doc "Returns the path of the application's priv directory."
+  def priv_dir(igniter, subpath \\ []) do
```

Comment:
Can you add an `@spec priv_dir(Igniter.t(), [String.t()]) :: String.t()`

Reply:
Yes, of course. The above functions were missing, so I didn't include them. They’ve been added now.

Reply:
🙇

Reply:
Yeah some don't have it but I'm trying to be more rigorous in having specs for everything 😓 Thank you!